### PR TITLE
Bugfix/nek cypress fixes

### PR DESCRIPTION
**Pull request by NEwertKrocker**
## What (if any) features are you implementing?
- Wrote a new test spec that stubs the async functions and populated it with dummy data ('Africa, Abridged').
## What (if anything) did you refactor?
- Nothing!
## Were there any issues that arose?
- I still don't think there's any way to test the game logic entirely automatically without exposing data to the front-end user. I guess this is why games have big QA teams!
## Is there anything that you need from your teammate?
- If you make major changes to the repo and you notice the test specs breaking, let me know, and I'll bring the Cypress files up to speed. 
## Any other comments, questions, or concerns?
-

### CODE REVIEW

**Comment by annamkummer** [comment]
Could we add tests to check the winStatus messages? If the user has a correct Bingo, if they have squares checked that were not called, and if they don't have four in a row?

**Comment by NEwertKrocker** [comment]
> Yes! The South America test (the non-stubbed one) already tests for a successful bingo, but I'll add tests for the other two.

File: cypress/integration/south_america_spec.js [hunk 1]
```diff
@@ -42,26 +55,43 @@ describe('South America game', () => {
...
+    cy.get('.bingo-btn').click();
+    cy.get('.square').contains('Peru').click();
+    cy.get('.square').contains('Argentina').click();
+    cy.get('.square').contains('Chile').click();
+    cy.get('.bingo-btn').click();
+    cy.get('.modal-content').contains('Keep playing')
+  })
+  it('should show a modal when the player runs out of coordinates', () => {
+    cy.get('.bingo-btn').click();
+    for (let i = 0; i < 17; i++){
```
Comment (jkiernan12) [inline cypress/integration/south_america_spec.js:84]: A lot better than having 17 clicks LOL 👍

**Review by jkiernan12 — APPROVED** [review]
Looks great -- nice job on this!!!!

**Comment by annamkummer** [comment]
Looks great!

**Pull request merged**